docs/machineset-nutanix.md

```diff
@@ -0,0 +1,63 @@
+# Creating a Nutanix Windows MachineSet
+
+_\<windows_server_image_name\>_ should be replaced with the full name of an image of a [supported version](wmco-prerequisites.md#supported-windows-server-versions) of Windows, pre-uploaded to the Prism-Central/Prism-Element where the Machine VMs will be created with.
```

Comment:
I'm assuming that "Windows" on line 3 refers to a Microsoft Windows OS and not the Windows Machine Config Operator. To avoid any potential confusion, consider replacing "of Windows" with "of the Windows operating system".

Reply:
Changed according to the suggestion.